pkg/cmd/kafka/create/create.go

```diff
@@ -397,6 +403,9 @@ type promptAnswers struct {
 func promptKafkaPayload(opts *options, userQuotaType accountmgmtutil.QuotaSpec) (*kafkamgmtclient.KafkaRequestPayload, error) {
 	f := opts.f
 
+	accountIDNullable := kafkamgmtclient.NullableString{}
+	cloudProviderIDNullable := kafkamgmtclient.NullableString{}
```

Comment:
I think this is very confusing. Cloud Provider should not be used as name as we have cloud provider as deployment target. Marketplace Provider could be better name. 

Marketplace feature is used for billing. Cloud Provider is deployment target. 
We can use AWS marketplace to pay for deployments for azure cloud provider and some region

Reply:
+1